src/runtime/step_runner.js

```diff
@@ -8,7 +8,7 @@ const {beginTiming, endTiming} = Time
 
 async function run({attachmentManager, defaultTimeout, scenarioResult, step, stepDefinition, transformLookup, world}) {
   beginTiming()
-  const parameters = stepDefinition.getInvocationParameters({scenarioResult, step, transformLookup})
+  const parameters = await Promise.all(stepDefinition.getInvocationParameters({scenarioResult, step, transformLookup}))
```

Comment:
Why is `Promise.all` needed? Does `getInvocationParameters` returns a promise or just the value? I'd prefer `getInvocationParameters` always returns a promise.